dotcom-rendering/window.guardian.d.ts

```diff
@@ -16,7 +16,6 @@ declare global {
 	 */
 	interface Window {
 		guardian: {
-			mustardCut: boolean;
```

Comment:
We're still setting this value so I think we need the type as well

Reply:
Back in!